WebContent/WOH-addRestaurant.jsp

```diff
@@ -0,0 +1,102 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+    pageEncoding="ISO-8859-1"%>
+<!DOCTYPE html>
+
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+
+<html>
+
+<head>
+	<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+	
+	<!-- Latest compiled and minified CSS -->
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+	
+	<!-- jQuery library -->
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+
+	<!-- Latest compiled JavaScript -->
+	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+	<script src="addRestaurant/addRestaurant.js"></script>
+	
+	<link rel="stylesheet" type="text/css" href="addRestaurant/WOH-addRestaurant.css">
+	<link rel="stylesheet" type="text/css" href="site/WOH-site.css">
+	
+	<title>Wheel of Hunger - Add Restaurant</title>
+</head>
+
+<body>
+<%@include file="banner/WOH-banner.html" %>
+	
+	<div class ="container">
+		<div class="row">
+			<div class="col-sm-10 col-sm-offset-1">
+				<form action = "" method = "POST" class="form-horizontal WOH-addRestaurant-form" id="">
+										
+					<!-- Restaurant Name Textbox -->
+						<div class="form-group">
+						<label class="control-label col-sm-2" for="WOH-addRestaurants-restaurantNameInput">Restaurant Name</label>
+							<div class="col-sm-10">
+	         					<input type = "text" class="form-control WOH-addRestaurant-restaurantNameInput" name = "name">
+	         				</div>
+	         			</div>
+	         			
+					<!-- Average Price Textbox -->
+						<div class="form-group">
+						<label class="control-label col-sm-2" for="WOH-addRestaurant-priceInput">Price Per Meal</label>
+							<div class="col-sm-10">
+	         					<input type = "text" class="form-control WOH-addRestaurant-priceInput" name = "price">
+	         				</div>
+	         			</div>
+					
+					<!-- Distance Textbox(If we use this) -->
+						<div class="form-group">
+						<label class="control-label col-sm-2" for="WOH-addRestaurant-distanceInput">Distance</label>
+							<div class="col-sm-10">
+	         					<input type = "text" class="form-control WOH-addRestaurant-distanceInput" name = "distance">
+	         				</div>
+	         			</div>
+	         			
+	         		<!-- Food Type Dropdown -->
+						<div class="form-group">
+	  					<label class="control-label col-sm-2" for="WOH-addRestaurant-typeInput">Select Food Type</label>
```

Comment:
Wrap this label in a div with the class col-sm-2 and remove this class from the label